AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts

```diff
@@ -93,7 +93,8 @@ export class SummaryComponent implements OnInit, OnDestroy {
         this.retrieveHearingSummary();
         this.switchOffRecording = this.recordingGuardService.switchOffRecording(this.hearing.case_type);
         this.interpreterPresent = this.recordingGuardService.mandatoryRecordingForHearingRole(this.hearing.participants);
-        this.hearing.audio_recording_required = this.interpreterPresent ? true : this.hearing.audio_recording_required;
+        this.hearing.audio_recording_required =
+            this.interpreterPresent && this.caseType !== 'Court of Appeal Criminal Division' ? true : this.hearing.audio_recording_required;
```

Comment:
do we have only the caseType name or we have an ID for the case type? I am concern about adding a hard coded string instead of using an ID.

Reply:
@marcogagliardi Hi Marco, at this point we only have caseType as a string. I looked to see if an ID was available, but there doesn’t appear to be. CaseType is used as a string in other parts of the code, such as RecordingGuardService - ClientApp\src\app\services\recording-guard.service.ts Let me know your thoughts, Andrew